cmd/satellite/main.go

```diff
@@ -1,28 +1,43 @@
 package main
 
 import (
-	"encoding/json"
-	"fmt"
-	"github.com/gpuctl/gpuctl/internal/status/handlers"
+	"log/slog"
 	"time"
+
+	"github.com/gpuctl/gpuctl/internal/femto"
+	"github.com/gpuctl/gpuctl/internal/uplink"
 )
 
 func main() {
-	// Just get the GPU data and print it in JSON
-	gpuHandler := handlers.NvidiaGPUHandler{}
-	for {
-		res, err := gpuHandler.GetGPUStatus()
-		if err != nil {
-			fmt.Printf("Failed to parse: %v\n", err)
-			return
-		}
 
-		ser, err := json.Marshal(res)
+	log := slog.Default()
+
+	s := satellite{
+		// TODO: Make this configurable
+		gsAddr: "http://localhost:8080",
+	}
+
+	log.Info("Starting satellite")
+
+	for i := 0; i < 10; i++ {
```

Comment:
We probably want this to run as a coroutine in the background while the satellite does other work.